src/sagemaker/jumpstart/artifacts/script_uris.py

```diff
@@ -90,3 +90,45 @@ def _retrieve_script_uri(
     script_s3_uri = f"s3://{bucket}/{model_script_key}"
 
     return script_s3_uri
+
+
+def _model_supports_inference_script_uri(
```

Comment:
Are these changes covered in UTs here: tests/unit/sagemaker/jumpstart/test_artifacts.py ?

Reply:
We do not unit test the internal functions within the `artifacts` directory. However, we have a whole suite of tests that involve these functions, ensuring no regression occurs. I figured it would be pedantic to test every internal function